ui/widgets/nuxeo-checkmark.js

```diff
@@ -54,11 +54,12 @@ import '../nuxeo-icons.js';
             height: var(--nuxeo-checkmark-height, 18px);
             cursor: pointer;
             border-radius: 50%;
-            border: 2px solid var(--nuxeo-checkmark-border-color, var(--nuxeo-border, gray));
+            border: 2px solid var(--nuxeo-checkmark-border-color, var(--nuxeo-text-default, gray));
             background-color: var(--nuxeo-checkmark-background-color, transparent);
             color: var(--nuxeo-icon-color, transparent);
             padding: 0;
             margin: 0;
+            opacity: 0.7;
```

Comment:
Looks too dark in the UI, 3:1 ratio would be enough as mentioned in https://jira.nuxeo.com/browse/ELEMENTS-1554

Reply:
opacity less than 7 doesn't work here.
![image](https://user-images.githubusercontent.com/109743453/208051955-fed4c4d7-7eaa-48e1-b4de-53c9248a0748.png)